Yank/commands/analyze.py

```diff
@@ -37,6 +41,12 @@
 Free Energy Required Arguments:
   -s=STORE, --store=STORE       Storage directory for NetCDF data files.
 
+Free Energy Optional Arguments:
+  --skipunbiasing               Skip the radially-symmetric restraint unbiasing. This can be an expensive step.
+  --distcutoff=DISTANCE         The restraint distance cutoff (in angstroms) to be used to unbias the restraint.
```

Comment:
What's the default behavior if `--skipunbiasing` isn't specified? Is it a distance- or energy-based unbiasing cutoff?

Reply:
Ah! Right, I still have to update the CLI docs. See above:

> By default the distance cutoff is computed automatically as the 99.9-percentile of the distance distribution at the lambda=1 state.

This is only if the user doesn't provide any distance/energy cutoff.

Reply:
Awesome, thanks!